src/Accelerators/NNPA/Conversion/ONNXToZHigh/ONNXToZHigh.cpp

```diff
@@ -101,6 +101,28 @@ Value getLSTMGRUGetY(
   return val;
 }
 
+Value getLSTMGRUGetYWithSequenceLens(Location loc, PatternRewriter &rewriter,
+    Value val, Value resY, Value sequenceLens, Value initialH) {
+
+  Value noneValue;
+  if (isNoneValue(resY)) {
+    return noneValue;
+  }
+
+  if (isNoneValue(sequenceLens))
+    return getLSTMGRUGetY(loc, rewriter, val, resY);
+
+  // Create a CustomOp to fix Y because zDNN GRU ignores the sequence_lens
+  std::vector<Value> inputs = {val, sequenceLens, initialH};
+  ONNXCustomOp customOp =
+      rewriter.create<ONNXCustomOp>(loc, resY.getType(), inputs);
+  StringAttr funcNameAttr = rewriter.getStringAttr("FixGRUY");
+  customOp->setAttr("function_name", funcNameAttr);
+  StringAttr shapeInferAttr = rewriter.getStringAttr("SameAs");
+  customOp->setAttr("shape_infer_pattern", shapeInferAttr);
+  return customOp.getResults()[0];
```

Comment:
Can we provide a new ZHigh Op, e.g. `ZHighLSTMGRUGetYWithSequenceLens`, instead of calling ONNXCustomOp here? The reason is that this fix is for NNPA only, and it's weird to expose the fix inside the lowering of CustomOp.

Reply:
Sure. CustomOp is a quick way to prototype. I will fix the bidirectional first.